lib/trln_argon/engine.rb

```diff
@@ -66,7 +65,9 @@ def initialize
           'url=http://unc.summon.serialssolutions.com/search?'\
           's.secure=f&s.ho=t&s.role=authenticated&s.ps=20&s.q='
         @contact_url = 'https://library.unc.edu/ask/'
-        @citation_formats = 'apa, mla, chicago, harvard, turabian'
+        # citation formats are coming from the citation-style-language/styles repository on GitHub
+        # (https://github.com/citation-style-language/styles/tree/b2be5aeeee7f00fd2032ac1daad995bbe95398cf)
```

Comment:
I would clarify the comment here; I found it a bit confusing. Indicate how an implementer can customize the citations listed via local env config. If I understand correctly, the _possible_ values one can choose from are defined in `.csl` files in that GitHub repo. Probably also don't need the full digest in that URL; I recommend changing the link to https://github.com/citation-style-language/styles/

Reply:
At lines 48 and 49, these configuration variables are removed but they are still referenced in `config/initializers/trln_argon.rb` (lines 27 and 28 in the latter)